hazelcast/src/test/java/com/hazelcast/internal/serialization/impl/compact/CompactStreamSerializerTest.java

```diff
@@ -57,21 +58,49 @@
 import java.time.OffsetDateTime;
 import java.time.ZoneOffset;
 import java.util.ArrayList;
+import java.util.function.Supplier;
 
 import static com.hazelcast.internal.serialization.impl.compact.CompactTestUtil.createCompactGenericRecord;
+import static com.hazelcast.internal.serialization.impl.compact.CompactTestUtil.createFixedSizeFieldsDTO;
 import static com.hazelcast.internal.serialization.impl.compact.CompactTestUtil.createMainDTO;
+import static com.hazelcast.internal.serialization.impl.compact.CompactTestUtil.createVarSizedFieldsDTO;
 import static com.hazelcast.nio.serialization.genericrecord.GenericRecordBuilder.compact;
 import static example.serialization.HiringStatus.HIRING;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
 import static org.junit.Assert.assertArrayEquals;
 import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertNull;
 import static org.junit.Assert.assertFalse;
 import static org.junit.Assert.assertTrue;
 
 @RunWith(HazelcastParallelClassRunner.class)
 @Category({QuickTest.class, ParallelJVMTest.class})
 public class CompactStreamSerializerTest {
 
-    SchemaService schemaService = CompactTestUtil.createInMemorySchemaService();
+    private final SchemaService schemaService = CompactTestUtil.createInMemorySchemaService();
+
+    private InternalSerializationService createSerializationService() {
+        return new DefaultSerializationServiceBuilder()
+                .setSchemaService(schemaService)
+                .setConfig(new SerializationConfig())
+                .build();
```

Comment:
```suggestion
        return createSerializationService(new CompactSerializationConfig());
```
Also, why does this return `InternalSerializationService`, whereas others return `SerializationService`?

Reply:
Moved and unified every `createSerializationService` methods in CompactTestUtil and removed all DefaultSerializationServiceBuilder usages except in `createSerializationService`